terraform/workspace-variables/productiondata.tfvars.json

```diff
@@ -13,5 +13,8 @@
   "paas_web_app_instances": 2,
   "paas_web_app_memory": 1536,
   "paas_worker_app_instances": 0,
-  "paas_worker_app_memory": 512
+  "paas_worker_app_memory": 512,
+  "azure_tempdata_storage_account_name": "s121p01registerprdatatmp",
+  "azure_resource_group_name": "s121p01-reg-prdata-rg",
```

Comment:
Is this `prdata` as in Pull Request or Production?  Might be better to use `pd` as this is the usual abbreviation for Production on Teacher Services

Reply:
@saliceti These resources already exist. If we are keen, we could delete them and align them with the naming?